containers/Makefile

```diff
@@ -1,2 +1,2 @@
 docker:
-	docker build .
+	./build.sh
```

Comment:
Script should be renamed to `docker_build.sh` (since we'll add a Singularity recipe too).

But if we have a script, the `Makefile` isn't of much use...

Either we should shove all the commands in the `Makefile`, or keep the script and drop the `Makefile`.

Reply:
I tried to put the commands in a makefile syntax but got pi**ed - so I choose the easy way out.

Reply:
Fine by me to just use scripts, my editor is also configured to expand tabs, so `Makefile` are kinda annoying (although I like their simplicity in use, and they're easy to debug, as opposed to some other tools who-shall-not-be-named).